src/main/resources/openapi.yml

```diff
@@ -1807,77 +1807,74 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RestError'
+                $ref: '#/components/schemas/ProblemDetail'
         404:
           description: User not found.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RestError'
+                $ref: '#/components/schemas/ProblemDetail'
         500:
           description: Server error.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RestError'
+                $ref: '#/components/schemas/ProblemDetail'
 components:
   schemas:
-    RestError:
-      title: REST Error
+    ProblemDetail:
+      title: Problem Detail
       description: The schema for all error responses.
       type: object
       properties:
+        type:
+          title: Type
+          description: Full URL that originated the error response.
+          type: string
+          format: uri
+          example: 'http://localhost:9966/petclinic/api/owner'
+          readOnly: true
+        title:
+          title: Title
+          description: The short error title.
+          type: string
+          example: NoResourceFoundException
+          readOnly: true
         status:
           title: Status
-          description: The HTTP status code.
+          description: HTTP status code
           type: integer
           format: int32
-          example: 400
-          readOnly: true
-        error:
-          title: Error
-          description: The short error message.
-          type: string
-          example: Bad Request
+          minimum: 400
+          maximum: 600
+          exclusiveMaximum: true
+          example: 500
           readOnly: true
-        path:
-          title: Path
-          description: The path of the URL for this request.
+        detail:
+          title: Detail
+          description: The long error message.
           type: string
-          format: uri
-          example: '/api/owners'
+          example: 'No static resource api/owner.'
           readOnly: true
         timestamp:
           title: Timestamp
           description: The time the error occurred.
           type: string
           format: date-time
-          example: '2019-08-21T21:41:46.158+0000'
-          readOnly: true
-        message:
-          title: Message
-          description: The long error message.
-          type: string
-          example: 'Request failed schema validation'
+          example: '2024-11-23T13:59:21.3820407'
```

Comment:
Time offset is mandatory in the date-time format (which uses https://www.rfc-editor.org/rfc/rfc3339#section-5.6 ), so I believe this should be something like '2024-11-23T13:59:21.3820407Z' or '2024-11-23T13:59:21.3820407+00:00'

Reply:
> O deslocamento de tempo é obrigatório no formato de data e hora (que usa https://www.rfc-editor.org/rfc/rfc3339#section-5.6 ), então acredito que deveria ser algo como '2024-11-23T13:59:21.3820407Z' ou '2024-11-23T13:59:21.3820407+00:00'

Hello @pvdbosch, I could implement it without any problems